<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KittiVisualizerQt</class>
 <widget class="QMainWindow" name="KittiVisualizerQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>503</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitterVertical">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitterHorizontal">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_sliders">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_dataSet">
           <item>
            <widget class="QLabel" name="label_dataSet">
             <property name="text">
              <string>Data set:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slider_dataSet">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_frame">
           <item>
            <widget class="QLabel" name="label_frame">
             <property name="text">
              <string>Frame:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slider_frame">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_tracklet">
           <item>
            <widget class="QLabel" name="label_tracklet">
             <property name="text">
              <string>Tracklet:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slider_tracklet">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_checkBoxes">
         <item>
          <widget class="QCheckBox" name="checkBox_showFramePointCloud">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Show frame point cloud</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_showTrackletBoundingBoxes">
           <property name="text">
            <string>Show tracklet bounding boxes</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_showTrackletPointClouds">
           <property name="text">
            <string>Show tracklet point clouds</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_showTrackletInCenter">
           <property name="text">
            <string>Show tracklet in center</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QVTKWidget" name="qvtkWidget_pclViewer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QComboBox" name="viewComboBox"/>
      <widget class="KittiImage" name="imageWidget" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
  <customwidget>
   <class>KittiImage</class>
   <extends>QWidget</extends>
   <header>KittiImage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>slider_dataSet</tabstop>
  <tabstop>slider_frame</tabstop>
  <tabstop>slider_tracklet</tabstop>
  <tabstop>checkBox_showFramePointCloud</tabstop>
  <tabstop>checkBox_showTrackletBoundingBoxes</tabstop>
  <tabstop>checkBox_showTrackletPointClouds</tabstop>
  <tabstop>checkBox_showTrackletInCenter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
